src/js/tab.js

```diff
@@ -0,0 +1,10 @@
+const locationHash = window.location.hash;
+
+if (locationHash) {
+  const tabsList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tab"]'));
+  const matchedTabs = tabsList.filter(tab => tab.hash === locationHash);
+
+  matchedTabs.map(tab => {
+    new bootstrap.Tab(tab).show();
+  });
```

Comment:
After this iteration, if `matchedTabs` have any elements will be good to remove hash from location to avoid confusion.

@codecalm what do you think?